src/xmpp_stream_pkix.erl

```diff
@@ -1,271 +0,0 @@
-%%%-------------------------------------------------------------------
```

Comment:
were these files deleted on purpose?

Reply:
Yes -- these files are already included in the dependencies for the project so I'm not quite sure why there were here, but they prevented generic xmpp clients from connecting (locally) due to some missing functions, etc, that the dependency versions do have so I removed them in order to test. Since I did this just to test, we could leave them in to be safe. Although at some point I'd like to get the project to a state where it fully works as an ejabberd server locally for easier development but that doesn't have to be this release. Let me know what you think.

Reply:
Yes, they are included in the dependencies so I'm not sure why they are here as well. These versions appear to be older than what the dependencies have for them and I encountered errors when trying to connect a generic xmpp client locally so I removed them to fix it so I could test locally. I'd like to get the project to a point where you can just check it out and start it and ejabberd fully works locally, but maybe that should be a separate thing let me know what you think.